src/coreclr/vm/dllimportcallback.cpp

```diff
@@ -607,7 +607,6 @@ bool TryGetCallingConventionFromUnmanagedCallersOnly(MethodDesc* pMD, CorInfoCal
     if (nativeCallableInternalData)
     {
         namedArgs[0].InitI4FieldEnum("CallingConvention", "System.Runtime.InteropServices.CallingConvention", (ULONG)(CorPinvokeMap)0);
-        namedArgs[0].InitI4FieldEnum("CallingConvention", "System.Runtime.InteropServices.CallingConvention", (ULONG)(CorPinvokeMap)0);
```

Comment:
Boo. What is that all about?

Reply:
I accidentally duplicated this line in another PR. Just removing the duplicate.